timely/src/dataflow/scopes/child.rs

```diff
@@ -97,6 +97,15 @@ where
 {
     fn name(&self) -> String { self.subgraph.borrow().name.clone() }
     fn addr(&self) -> Vec<usize> { self.subgraph.borrow().path.clone() }
+
+    fn addr_for(&self, index: usize) -> Vec<usize> {
```

Comment:
I think we should eventually add a comment here, or enrich the name to `addr_for_child` to be clear. Or `addr_plus` or .. idk something to be clear what is going on here. It's not a great mystery once you read the code, ofc. Don't worry about this if you don't want, and I can tidy in post.